Dockerfile.rhel

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/openshift/release:rhel-7-release-openshift-4.7 AS build
+FROM registry.ci.openshift.org/openshift/release:rhel-7-release-openshift-4.8 AS build
```

Comment:
It looks like this tag doesn't exist in the registry.ci.openshift.org registry. I know we had a problem with this before with the 4.7 equivalent of this PR, but that `registry.ci.openshift.org/openshift/release:rhel-7-release-openshift-4.7` worked after some manual intervention and the bot updated the PR.